src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+            cursor: {}
+
+- Name: ExpressionSqrt
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionSqrt
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {sqrt: { $sqrt: $posInt}}}]
+            cursor: {}
+
+- Name: ExpressionSwitch
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionSwitch
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {switch: { $switch: {branches: [{case: {$gte: [$int, 0]}, then: "GTE 0"}, {case: {$lt: [$int, 0]}, then: "LT 0"}], default: "Out of bounds"}}}}]
+            cursor: {}
+
+- Name: ExpressionTestToLower
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionToLower
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {toLower: { $toLower: $string}}}]
+            cursor: {}
+
+- Name: ExpressionTestToUpper
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionToUpper
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {toUpper: { $toUpper: $string}}}]
+            cursor: {}
+
+- Name: ExpressionRegexFind
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionRegexFind
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {regexFind: { $regexFind: {input: $string, regex: /(s(be)*)/}}}}]
```

Comment:
Nice sbe reference :D What strings does this match? does it match s, sbe, sbebe, and so on?

Reply:
Yes, however, I changed it now to match on a predetermined string